api/models/templates/app.tmpl

```diff
@@ -150,12 +145,18 @@
 {{ define "process-conditions" }}
   {{ range $k := .EntryNames }}
     "Blank{{ upper $k }}Service": { "Fn::Equals": [ "", "" ] },
+    "Blank{{ upper $k }}Cpu": { "Fn::Equals": [ { "Ref": "{{ upper $k }}Cpu" }, "-1" ] },
     "Disabled{{ upper $k }}DesiredCount": { "Fn::Not": [{ "Fn::Equals": [ {"Ref": "{{ upper $k }}DesiredCount" }, -1 ] }] },
   {{ end }}
 {{ end }}
 
 {{ define "process-params" }}
   {{ range $e := . }}
+    "{{ upper $e.Name }}Cpu": {
+      "Type": "Number",
+      "Default": "-1",
```

Comment:
Can we do `Default: 0` and always pass it through? I think your logic is correct as-is but not needing a Condition would be simpler.